impls/src/adapters/mod.rs

```diff
@@ -113,9 +115,13 @@ pub fn create_sender(
 			)
 			.into());
 		}
-		"file" => {
+		"file" | "qr" | "emoji" => {
```

Comment:
Would be nice if method was a enum here too

Reply:
This part of the code is just error handling. I don't know exactly what generates this error, nor have I tried to do this, but since it is based on the "method", if the Emoji or QR generated this error they would go to the option `_ => this method does not exist`. Which could cause confusion.

So as Emoji and QR are similar to the File method, I added the same error.